src/test/resources/org/mule/module/apikit/metadata/headers-in-raml08/api.raml

```diff
@@ -9,10 +9,6 @@ baseUri: https://www.example.com/api/
         headers:
           stringHeader:
             type: string
-            minLength: 2
-            maxLength: 10
-            enum: ["enum1", "enum2"]
-            default: "enum1"
```

Comment:
Why we remove this from the tests?

Reply:
because when serializing this, order of the annotations cannot be assured, so we endup with different serialized models and our tests fail, Also, we don't need to test this particular example, it's more like a n integration test